metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/PartitionSequenceIds.java

```diff
@@ -13,22 +13,24 @@
  *     See the License for the specific language governing permissions and
  *     limitations under the License.
  */
-package com.netflix.metacat.connector.hive.util;
+package com.netflix.metacat.connector.hive.sql;
 
-import com.netflix.metacat.common.server.connectors.model.PartitionInfo;
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
 
 /**
- * PartitionDetail.
- * @author zhenl
- * @since 1.0.0
+ * Class representing the ids for a partition.
+ *
+ * @author amajumdar
  */
 @Data
+@NoArgsConstructor
 @AllArgsConstructor
-public class PartitionDetail {
-    private Long id;
-    private Long sdId;
+@EqualsAndHashCode(callSuper = false)
+public class PartitionSequenceIds {
```

Comment:
Is this class desired to be mutable or is it a DTO? As it stands right now it is mutable.

Reply:
Yes.